src/plugins/irc-events/whois.js

```diff
@@ -27,6 +27,9 @@ module.exports = function(irc, network) {
 				text: "No such nick: " + data.nick
 			});
 		} else {
+			// Absolute datetime in milliseconds since nick is idle
+			data.idleTime = Date.now() - data.idle * 1000;
```

Comment:
Not entirely sure if whois messages can have `server-time`, but if they do, use that instead maybe?